frontend/Styleguide/Icon.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { Icon, IconVariant } from "./Icon";
+
+describe("<Icon />", () => {
+  it(`always renders a <title>`, () => {
+    Object.keys(IconVariant).forEach((iconVariant) => {
+      const w = render(<Icon variant={iconVariant} />);
```

Comment:
What does the w stand for?

Reply:
"wrapper." In the testing library docs, it encourages destructuring what you need rather than storing the whole wrapper object:

```js
const { baseElement } = render(...)
```

However, I prefer to store the whole wrapper object so i don't have to go up and change the destructuring when I want to use a new method. I can just do `w.baseElement` and `w.rerender()` without fixing up the destructuring. It's a small thing.

The word wrapper originates from enzyme, which is the library that `@testing-library/react` competes with and replaces. So I'm stealing the terminology from there since the testing library docs don't have a word for the wrapper object, as far as i know